styles/styles.css

```diff
@@ -0,0 +1,389 @@
+body {
+  margin: 0;
+  width: 100%;
+}
+
+/*
+header {
+  background-color: #0A3C61;
+  height: 21rem;
+  text-align: left;
+  display: flex;
+  justify-content: center;
+}
+*/
+header {
+  background-color: #0A3C61;
+  height: 24rem;
+  justify-content: flex-start;
+
+}
+
+header h1 {
+  color: black;
+  position: absolute;
+  left: 13rem;
+  top: 1rem;
+  font-size: 3rem;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 1.5rem;
+  font-family: sans-serif;
+  font-style: italic;
+  line-height: 2rem;
+}
+
+.b-box {
+  width: 30rem;
+  height: 3rem;
+}
+
+/*
+.header-img {
+  object-fit: cover;
+  width: 100%;
+  height: inherit;
+  object-position: top;
+}
+*/
+
+.header-img {
+  width: 100%;
+  height: 24rem;
+  background-color: #0066ff;
+  background-image: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.ftrlogo {
+  height: 30%;
+}
+
+.b2box {
+  height: 70%;
+  width: 40%;
+  margin: 4%;
+  min-width: 227px;
+  max-width: 484px;
+}
+
+.hdrtxt {
+  justify-content: center;
+}
+
+.ckout {
+  margin: 4%;
+}
+
+.art1 {
+  width: 60%;
+  display: inline-block;
+  margin: 10%;
+}
+
+
+
+
+
+
+/* Style the links inside the navigation bar */
+nav {
+  font-family: sans-serif;
+}
+
+nav li {
+  padding: 0 5px;
+  position: relative;
+  left: 11rem;
+}
+
+nav li a:after{
+  content: " / ";
+  position: absolute;
+  right: 0;
+  top : 15px;
+}
+
+nav li:last-child a:after{
+    display: none;
+}
+
+nav ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  height: 50px;
+  overflow: hidden;
+  background-color: #EBEBEB;
+}
+
+nav li {
+  display: inline-block;
+}
+
+nav li a {
+  display: block;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+/* Change the link color to #111 (black) on hover */
+nav li a:hover {
+  background-color: Grey;
+}
+
+.wrapper {
+  width: 80%;
+  min-height: auto;
+  margin: 2rem auto;
+  overflow: hidden;
+  line-height: 1.5rem;
+  font-size: 1rem;
+  font-family: sans-serif;
+  display: flex;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 59%;
+}
+
+@media screen and (min-width: 768px){
+  .flex-container{
+    display: flex;
+    width: 75%;
+  }
+}
+
+
+
+.flex-container3 {
+  display: none;
+  width: 0;
+  padding: 1rem;
+}
+
+@media screen and (min-width: 768px){
+  .flex-container3{
+    display: block;
+    width: 25%;
+  }
+}
+
+.smdlink {
+  clear: both;
+}
+
+.dogimg {
+  width: 100%;
+}
+
+.div1 {
+  width: 98%;
+  display: grid;
+}
+
+.div2 {
+  width: 30%;
+  min-width: 275px;
+  box-sizing: border-box;
+  border: 4px solid #A9A9A9;
+  display: inline-block;
+  margin: 5px 5px;
+  padding: 1rem;
+}
+
+.div3 {
+  width: 52%;
+  height: auto;
+  min-width: 275px;

```

Comment:
This doesn't quite work. The min-width will overflow. I'd take that off and have teh width be at 100%. Maybe increase the width of your sidebar from 25% to 33% or somethign to keep it from getting too narrow.